site/src/pages/CreateTemplatePage/CreateTemplateForm.tsx

```diff
@@ -728,8 +728,8 @@ const DefaultTTLHelperText = (props: { ttl?: number }) => {
 
   return (
     <span>
-      Workspaces will default to stopping after {ttl} {hours(ttl)} without
-      activity.
+      Workspaces will default to stopping after {ttl} {hours(ttl)}. This will be
+      extended by 1 hour after last activity in the workspace was detected.
```

Comment:
```suggestion
      extend by 1 hour intervals if the workspace is being used.
```
Thoughts on rephrasing this like this. 

The current rephrasing still makes me confused.